test/no-content.html

```diff
@@ -17,22 +17,19 @@
     <test-fixture id="juicy-html-with-href">
         <template>
             <!-- nest to workaround test-fixture bug -->
-            <div><template is="juicy-html" href="./mock/smth">
-            </template></div>
+            <div><juicy-html href="./mock/smth"></juicy-html></div>
```

Comment:
I know this isn't from this PR, but this should be [`mock.html`](https://github.com/Juicy/juicy-html/blob/master/test/mock/smth.html), it's logging 404.